crates/walrus-service/src/event/event_processor/processor.rs

```diff
@@ -147,13 +176,23 @@ impl EventProcessor {
 
         let clients = client_manager.into_client_set();
 
+        let (coordination_state, _coordination_rx) = CatchupCoordinationState::new();
+        let coordination_state = Arc::new(coordination_state);
+
         let catchup_manager = EventBlobCatchupManager::new(
             event_processor.stores.clone(),
             clients,
             system_config,
             runtime_config.db_path.join("recovery"),
             metrics_registry,
+            catchup::CatchupRuntimeConfig {
+                coordination_state: coordination_state.clone(),
+                coordination_timeout: Duration::from_secs(STARTUP_CATCHUP_TIMEOUT_SECS),
+                processing_timeout: Duration::from_secs(STARTUP_CATCHUP_TIMEOUT_SECS),
+            },
         );
+        coordination_state.notify_tailing_stopped();
```

Comment:
question: do we need this call here? Tailing is not started at this point. Also, IIUC, this coordination state is a short lived coordination state, so can we document that this is actually not needed/used during initial catchup?

Reply:
Oh, I think I know why this is needed. In catchup_using_event_blobs, it is waiting for tailing to stop.

I feel it's cleaner if we initialize coordination_state with `is_tailing_stopped` set to true. But not too strongly.

Ideally, if there is only one stance of CoordinationState, the one that used in the long running monitor, it's easier to read the code.

In general, I feel the interaction between EventBlobCatchupManager and CoordinationState requires someone to know the code very well to reason about.

Reply:
done, there is only one instance of coordination now.